lnrpc/rpc.proto

```diff
@@ -222,9 +222,17 @@ message WalletBalanceResponse {
     double balance = 1;
 }
 
+message RTChannel {
```

Comment:
Naming suggestion: `RoutingTableLink`

What does `edgeID` represent in this case? The outpoint (txid:index) of the channel itself? 

Also I think capacity should be an `int64` rather than `double`.